src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawable {
```

Comment:
do we need to implement interfaces in child figure classes?

Reply:
done, no, we don't need implement interfaces in child figure classes, because parent class Figure, already implement interface